frontend/src/App.css

```diff
@@ -44,3 +44,13 @@
 .read-the-docs {
   color: #888;
 }
+
+html.dark .svg-display {
+  filter: invert() invert(19%) sepia(9%) saturate(1857%) hue-rotate(180deg)
+    brightness(99%) contrast(87%);
```

Comment:
What a filter chain. First of all:

```diff
-  filter: invert() invert(19%) sepia(9%) saturate(1857%) hue-rotate(180deg)
-    brightness(99%) contrast(87%);
+  filter: invert(81%) sepia(9%) saturate(1857%) hue-rotate(180deg) contrast(87%);
```

Second, the heavy over-saturation of 18.57 really messes with the colors, which makes it hard to see e.g. which box is marked with the bold outline for context diagrams. At this point maybe we should also rethink the general idea, and instead of inverting the brightness values, only add a simple filter like this to tone it down a bit:

```diff
-  filter: invert() invert(19%) sepia(9%) saturate(1857%) hue-rotate(180deg)
-    brightness(99%) contrast(87%);
+  filter: contrast(1.2) brightness(.8);
```

What do you think about this?

Reply:
Just toning it down still leaves it white. Especially when you click to enlarge the SVG, there will be an awful lot of white on your screen for a dark mode.